include/mbgl/ios/MGLMapView.h

```diff
@@ -280,6 +281,12 @@ IB_DESIGNABLE
 /** Resets the map to the minimum zoom level, a center coordinate of (0, 0), and a northern heading. */
 - (void)resetPosition;
 
+/** Triggers another render pass even when it is not necessary. */
+- (void)invalidate;
+
+/** Returns whether the map view is currently loading or processing any assets required to render the map */
+- (BOOL)isFullyLoaded;
+
```

Comment:
How can I make these available for the Benchmark application, but not for other applications?